docs/docs/external/getting-started.md

```diff
@@ -85,6 +85,16 @@ To build the library, do the following:
   make build-local
   ```
 
+### Sample Fabric Application Contract (Chaincode)
+
+To prepare this contract (chaincode) for deployment (see next section):
```

Comment:
For deployment of this chaincode in fabric-network, these steps are not required.
These steps are only required if you want to test the chaincode or build it manually.

Reply:
Removed.